packages/pluggableWidgets/popup-menu-web/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "popupmenu-web",
```

Comment:
The name of the packge does not align with the folder name!

Reply:
✅